wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+   - Stage 2: Early to Mid-game Wildcard (Short Duration)
+   - Stage 3: Pure Density
+   - Stage 4: Mid-game Technical
+   - Stage 5: Late-game Releases
+   - Stage 6: Mid to Late-game Wildcard (Long Duration)
+   - Stage 7: Speed Density
+4. This tournament will make use of the **Average Placement** method to seed players.
+5. In the qualification process, every player who signs up has to schedule for a lobby during the Qualfiers Stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
+6. After completing the scheduled lobby, each player will have 1-2 scores recorded for each of the 7 qualifier maps. For each of the maps, the higher score of the two is taken and is used to provide the player's placement for each individual map.
+7. After all players have completed their qualifier lobbies, each player's Average Placement will be obtained by dividing their Sum Placement by the number of maps in the qualifier mappool. This Average Placement will be used to cut down to the top 64 players, who will advance to the Bracket Elimination stage. Every other player is eliminated from the tournament.
+
+##### Bracket Matches
+
+1. After Qualifier Stage, the top 64 players will be arranged into a standard Double Elimination Bracket, where the highest seeded player will face the lowest seeded player, 2nd highest faces 2nd lowest, and so on. A player will be eliminated from the tournament after losing 2 matches in the bracket stage.
+
+### Tournament Procedure
+
+#### Pre-match procedure
+
+1. Players will be informed of their match by an assigned referee 30 minutes before their matches commence, through a ping on Discord.
+2. Players will be provided with an invite to the lobby hosted by the referee 5 minutes before their match commences. Players are required to be present in the lobby before the stipulated match time. 
+3. Warmups will not be allowed for this tournament.
+
+#### Mid-match procedure
+
+1. After both players have joined the lobby, both players will run the !roll command in the lobby chat.
+2. The player with the higher roll will pick first, and choose whether to protect first. If they choose to pick first, then they will ban second. The player with the lower roll receives the opposite of this.
+3. Following the ban/pick procedure, both players will alternate in picks, and play the picked map simultaneously. The player who obtains a higher score will win a point, and this process repeats until one of the players wins the required number of points to claim the win for the match.
+   - The required number of points to win the match varies for each stage.
+4. Each player receives the right to use a 2-minute timeout once per match. If players find the need to use the timeout between picks, they should raise their request to the referee facilitating their match.
+5. Players are encouraged to use their timeout only when an urgent situation or personal condition calls for it, and not to stall for time.
```

Comment:
I'd say both of which are the same, as in the urgency of the "personal condition" calls for a timeout.
```suggestion
5. Players are encouraged to use their timeout only when an urgent situation calls for it, and not to stall for time.
```